Vagrantfile

```diff
@@ -1,14 +1,21 @@
-is_jenkins = ENV['USER'] == 'jenkins'
+# Whether or not this is jenkins user
+IS_JENKINS = ENV['USER'] == 'jenkins'
+
+# Specify how much memory to use
+VAGRANT_MEMORY = (ENV['VAGRANT_MEMORY'] || '1024').to_i
+
+# Specify whether or not to enable Puppet debug output
+PUPPET_DEBUG = (ENV['VAGRANT_PUPPET_DEBUG'] || '0') == '1'
 
 Vagrant.configure("2") do |config|
   config.vm.box = "puppetlabs/centos-7.0-64-puppet"
 
   config.vm.provider "virtualbox" do |v|
     v.name = "socorro-vm"
-    v.memory = 512
+    v.memory = VAGRANT_MEMORY
   end
 
-  if not is_jenkins
+  if not IS_JENKINS
```

Comment:
I changed this to make it easier to pick out. I don't know if this is a proper convention, though.